README.md

```diff
@@ -1,41 +1,60 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/-ucQIGTc)
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-718a45dd9cf7e7f842a935f5ebbe5719a5e09af4491e668f4dbf3b35d5cca122.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15216106&assignment_repo_type=AssignmentRepo)
 # SE-Assignment-2
 Assignment: Introduction to Software Engineering
 Instructions:
 Answer the following questions based on your understanding of software engineering concepts. Provide detailed explanations and examples where appropriate.
 
 Questions:
 Define Software Engineering:
-
```

Comment:
Software engineering is the systematic approach to software development, operation, and maintenance. It includes applying engineering principles and practices during all stages of the software development process: requirements analysis, design, implementation, testing, deployment, and support of developed software. This is a way of producing properly working software and also answering user needs with high product quality, reliability, efficiency, and maintainability.